data/data-pipeline/data_pipeline/etl/score/schemas/datasets.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import List, Optional
+
+
+class FieldType(Enum):
+    STRING = "string"
+    INT64 = "int64"
+    BOOL = "bool"
+    FLOAT = "float"
+    PERCENTAGE = "percentage"
+    LOSS_RATE_PERCENTAGE = "loss_rate_percentage"
```

Comment:
Hmm. The only thing different about `LOSS_RATE_PERCENTAGE` from `PERCENTAGE` is that loss rate percentage is printed with more significant digits when we write the output to Excel. 

Maybe instead of doing this, we just add an optional new characteristic to the LoadFields called `number_of_decimals_in_output` or something and have it default to 2 for all floats?

Reply:
Great! Done!